packages/ingest/http.js

```diff
@@ -136,13 +137,24 @@ module.exports.httpMixin = (superclass) => class extends superclass {
     const s3uri = buildS3Uri(bucket, key);
     log.info(`Sync ${remoteUrl} to ${s3uri}`);
 
+    let headers = {};
+    try {
+      const headResponse = await got.head(remoteUrl);
+      headers = headResponse.headers;
+    }
+    catch (err) {
+      log.info(`HEAD failed for ${remoteUrl} with error: ${err}.`);
+    }
+    const contentType = headers['content-type'] || mime.lookup(key);
+
     const pass = new PassThrough();
     got.stream(remoteUrl).pipe(pass);
 
     await s3().upload({
       Bucket: bucket,
       Key: key,
-      Body: pass
+      Body: pass,
+      ContentType: contentType || null
```

Comment:
If `contentType` is set, use that. Otherwise we'll just send `null` and let aws set the default.